core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -401,6 +402,23 @@ class SparkContext(config: SparkConf) extends Logging {
     }
   }
 
+  /** Change logLevel of specific package or class name.
+   *  This overrides any user-defined log settings.
+   *
+   * @param loggerName package or class name such as "org.apache.spark" or
+   *                   "org.apache.spark.SparkContext"
+   * @param logLevel The desired log level as a string.
+   *                 Valid log levels include: ALL, DEBUG, ERROR, FATAL, INFO, OFF, TRACE, WARN
+   */
```

Comment:
Need the "since 4.0.0" on the new methods

Reply:
We need this in Pyspark, no?

Reply:
Added since. I'll add this in Pyspark in followup PR.